docs/docs/introduction/getting-started.md

```diff
@@ -0,0 +1,44 @@
+---
+id: getting-started
+title: Getting Started with electron-redux
+description: 'Introduction > Getting Started: Resources to get started with electron-redux'
+hide_title: false
+---
+
+Electron-Redux is an Redux Store Enhancer that helps you loosely synchronize the redux stores in multiple electron processes.
+
+When working with Electron, using Redux poses couple of problems, since main and renderer processes are isolated and IPC is a single way of communication between them. Electron-Redux, lets you to register all your Redux stores in the main & renderer process, and enable cross-process action dispatching & loose store synchronization.
```

Comment:
```suggestion
When working with Electron, using Redux poses a couple of problems, since main and renderer processes are isolated and IPC is the only way of communication between them. Electron-redux, allows you to register all your Redux stores in the main & renderer process, and enable cross-process action dispatching & loose store synchronization.
```